README.md

```diff
@@ -7,6 +7,9 @@ ioBroker dev-server is a simple command line tool running on Windows, Linux and
 
 **Note:** dev-server requires at least Node.js 14.
 
+Instead of manually editing the package.json file, users should simply run ```npm install``` to automatically install the dependencies listed in the package.json
```

Comment:
This is common sense for NodeJS developers. I wouldn't put this in the general section of this tool.

Reply:
I think it is ok to state that because many devs do not know tat they need to to that e.g. to update package-lock or such ... so not the worst idea tbh